_includes/manuals/1.24/1.2_release_notes.md

```diff
@@ -10,6 +10,28 @@ Foreman is now capable of performing Single Sign-On with the help of various Ope
 #### Debian 10 (Buster) support
 Packages for Foreman and its various plug-ins are now being built for Debian 10 (Buster) as well.
 
+#### Dynflow deployment with Sidekiq
+Foreman is now able to use Dynflow deployed with Sidekiq, as described in [Upcoming changes to Dynflow](https://community.theforeman.org/t/upcoming-changes-to-dynflow/14926). This new deployment model should address issues we have seen when restarting the Dynflow executor. This is currently included as a preview and can be enabled by hand, if desired.
+
+To enable it on EL7 and derivatives:
+
+```
+# Install and enable redis
+yum install -y redis
+systemctl enable --now redis
+
+# Install new Dynflow service files
+yum install -y foreman-dynflow-sidekiq
+
+# Disable old Dynflow deployment, the old and new can run at once, but it isn't advised
+systemctl disable --now dynflowd
+
+# Enable the new dynflow services
+systemctl enable --now dynflow-sidekiq@{orchestrator,worker}
+```
+
+On Debian based platforms this feature is not packaged yet, but it is planned for the near future.
```

Comment:
I would move this to the main block above, so debian users know right away not to try it.